pragma/tests/fetchers_test.py

```diff
@@ -60,15 +102,15 @@ async def test_async_fetcher(fetcher_config, mock_data):
 
 
 @pytest.mark.asyncio
-async def test_async_fetcher_404_error(fetcher_config):
-    with aioresponses() as mock:
+async def test_async_fetcher_404_error(mocker, fetcher_config, forked_client):
+    with aioresponses(passthrough=[forked_client.url]) as mock:
```

Comment:
Same as the previous test, we only want to mock the external fetcher APIs and not the RPC.